src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -107,16 +108,16 @@
     private String translationRequired = EMPTY;
     private String translationProvided = EMPTY;
 
-    @OneToMany(mappedBy = "item")
-    @Where(clause = "type = 'asl'")
+    @OneToMany(mappedBy = "item", cascade = CascadeType.ALL)
```

Comment:
Using `CascadeType.ALL` seems to be the only thing that worked here.  I tested using `CascadeType.PERSIST`, but that wasn't sufficient.  I could try `CascadeType.MERGE` here to see if that addresses the issue of `Attachment`s not being saved to the `item_attachment` table after being added to an item.

I suspect the underlying issue is that when I create the `Attachment`s and add them to the `BaseItem`, the `Attachment` objects are in a "detached" state from JPA's underlying entity manager.  I haven't confirmed that yet; that's conjecture (but seems fairly likely).  There's probably a better way to handle object creation, but this is the best I could come up with for now.